examples/proposal-list.yaml

```diff
@@ -81,8 +81,8 @@
     receiver: https://farm.example/
     resourceClassifiedAs: https://www.wikidata.org/wiki/Q4917 # US Dollar
     resourceQuantity:
-      qudt:unit: unit:Number
-      qudt:numericValue: 32
+      om2:hasUnit: om2:one
```

Comment:
s/b http://www.ontology-of-units-of-measure.org/resource/om-2/UnitedStatesDollar ?